### ISSUE TITLE
Freeze followed by a watch reboot when pressing back button to exit

### ISSUE THREAD

**Issue by Protonus** [kije/PebblePong-2#1]
Hi, I'm getting a freeze, followed by a Pebble reboot, when I try and press the back button after playing for a while (3+ score for instance).  I'm running 2.0 Beta 2 on both phone and watch, on Android 4.1.1.  Thanks.

**Comment by kije** [comment]
Thank you for your report. I'm about to push a fix for that.
I think this is also because of the beta status of the Pebble 2.0 firmware. 
On 2.0 Beta 1 I had the issue, where the app crashed "randomly", without obvious reason.
Also, on Beta 1 and 2, it crashed as I tried to implement the accelerometer for controlling the paddles. 

Try to upgrade to 2.0 Beta 4 and, as soon as I released the fix, install the newest version of PebblePong 2
